lib/generators/spree/retail/install/templates/config/initializers/spree_retail.rb

```diff
@@ -0,0 +1,4 @@
+# Spree Retail Setup
```

Comment:
Just a quick naming question here. Since this exact gem will be called 'solidus_retail' should the initializer have the same name?